common/services/person.js

```diff
@@ -2,8 +2,11 @@ const { mapValues, uniqBy } = require('lodash')
 
 const apiClient = require('../lib/api-client')
 
-function getFullname({ first_names: firstNames, last_name: lastName }) {
-  return `${lastName}, ${firstNames}`
+function transform(person) {
+  return {
+    ...person,
+    fullname: `${person.last_name}, ${person.first_names}`,
```

Comment:
is the comma in between the first and last name intentional?

Reply:
Yes, so that's the way our app will always display a "persons" name in this format. 

It was one that always surprised me during research but our users are used to seeing a persons last name first in all their other systems.